erpnext/hr/doctype/process_payroll/process_payroll.py

```diff
@@ -128,11 +126,9 @@ def create_submit_log(self, all_ss, not_submitted_ss):
 
 		submitted_ss = self.format_as_links(list(set(all_ss) - set(not_submitted_ss)))
 		if submitted_ss:
-			mail_sent_msg = self.send_email and " (Mail has been sent to the employee)" or ""
 			log = """
-			<b>Salary Slips Submitted %s:</b>\
-			<br><br> %s <br><br>
-			""" % (mail_sent_msg, '<br>'.join(submitted_ss))
+				<b>Salary Slips Submitted: %s</b>
```

Comment:
Did you see how the email looks?

It will now look like:

```
Salary Slips Submitted: SAL-1
SAL-2
SAL-3
```

so there should be a `<br>` after the colon and then `%s`